src/components/HeaderComponent.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="flex flex-row items-center justify-between">
```

Comment:
You don't need to add _flex-row_ class, because the flex direction is row by default